pkg/daemon/ceph/client/image.go

```diff
@@ -115,6 +115,23 @@ func DeleteImage(context *clusterd.Context, clusterName, name, poolName string)
 	return nil
 }
 
+func ResizeImage(context *clusterd.Context, clusterName, name, poolName, monitors, keyring string, size uint64) error {
+	imageSpec := getImageSpec(name, poolName)
+	args := []string{
+		"resize",
+		imageSpec,
+		fmt.Sprintf("--size=%s", strconv.FormatUint(size, 10)),
+		fmt.Sprintf("--cluster=%s", clusterName),
+		fmt.Sprintf("--keyring=%s", keyring),
+		"-m", monitors,
+	}
+	output, err := ExecuteRBDCommandWithTimeout(context, clusterName, args)
+	if err != nil {
+		return fmt.Errorf("failed to resize image %s in pool %s: %+v. output: %s", name, poolName, err, string(output))
```

Comment:
`err` should be the last thing displayed.

Reply:
Maybe We don't need the "output" here? But I cat only change the order.

Reply:
I picked up the same error structure as DeleteImage method above, but now they are different. Should I change other error structures in the client/image.go too?

Reply:
Yes please.